userspace/falco/app_actions/print_ignored_events.cpp

```diff
@@ -39,25 +39,26 @@ application::run_result application::print_ignored_events()
 	configure_interesting_sets();
 
 	/* Search for all the ignored syscalls. */
-	std::unique_ptr<sinsp> inspector(new sinsp());
-	std::unordered_set<uint32_t> all_ppm_sc = inspector->get_all_ppm_sc();
-	std::unordered_set<uint32_t> ignored_ppm_sc;
-
-	for(const auto& it : all_ppm_sc)
+	std::unordered_set<uint32_t> all_events;
+	for (uint32_t j = 0; j < PPM_EVENT_MAX; j++)
 	{
-		/* If the syscall is not in this set we ignore it. */
-		if(m_state->ppm_sc_of_interest.find(it) == m_state->ppm_sc_of_interest.end())
+		if (!sinsp::is_old_version_event(j)
+				&& !sinsp::is_unused_event(j)
+				&& !sinsp::is_unknown_event(j))
```

Comment:
These are the events we don't want to show to the user because are useless or misleading, in the next future we could also exclude some other internal events :thinking: